packages/server/src/logger.ts

```diff
@@ -11,36 +12,97 @@ import { CloudWatchLogger } from './util/cloudwatch';
 
 export enum LogLevel {
   NONE = 0,
```

Comment:
If you change `NONE` to 5, you can remove the special case.

Reply:
Although, to me, the previous numeric values made more sense, in the colloquial sense of "use a higher log level" == "log more stuff"

Reply:
Okay yeah that clicks for me — I'll switch this back